connections.go

```diff
@@ -160,9 +171,38 @@ func (s *ConnectionServiceOp) ProjectList(id string, opts *GetOptions) ([]Connec
 	return s.list(apiUrl, opts)
 }
 
-func (s *ConnectionServiceOp) Delete(id string) (*Response, error) {
+func (s *ConnectionServiceOp) Delete(id string, wait bool) (*Response, error) {
```

Comment:
@displague I implemented conditional wait for the conenction delete. It's necessary for cleanup - we can't remove the VLANs before the connection is gone, otherwise we get API errors.